release-checklist.md

```diff
@@ -47,3 +48,4 @@
 - [ ] `git branch -D master-test`
 - [ ] `git push origin :master-test`
 - [ ] Announce availability in #engineering and on the forum here: https://forum.blockstack.org/c/releases (Include a link to the changelog)
+- [ ] Announce availability to the community via Jenny Mith <jenny@blockstack.com> and Mitchell Cuevas <mitchell@blockstack.com>. Provide a succinct list of changes, the like to the changelog and latest release as well as a desired date for the community update. Consider doing an AMA with the community.
```

Comment:
Is the AMA meant to happen for every release?

Reply:
No, optional. I think this would require a judgement call on the releaser, ie "is this important enough to organize an AMA for"? Should I point that out as well?

Reply:
I think it should be clear when that should happen. i.e. Consider doing an AMA on major/breaking releases.

Reply:
Makes sense, adding that